Source/items.cpp

```diff
@@ -2101,22 +2101,22 @@ void SetupAllItems(int ii, int idx, int iseed, int lvl, int uper, int onlygood,
 
 void SpawnItem(int m, int x, int y, BOOL sendmsg)
 {
-	int ii, onlygood, idx;
+	int ii, idx;
+	BOOL onlygood = FALSE;

```

Comment:
Breaks bin exactness, along with oher changes in your pr. Not that I care im not part of this project, but something to consider.